test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -274,39 +274,87 @@ public void Should_Write_History_For_Audited_Entities_Update()
+                entityChange1.EntityId.ShouldBe(blog1Id.ToJsonString(false, false));
+                entityChange1.PropertyChanges.Count.ShouldBe(1);
+
+                var propertyChange1 = entityChange1.PropertyChanges.Single(pc => pc.PropertyName == nameof(BlogEx.BloggerName));
+                propertyChange1.OriginalValue.ShouldBe(originalValue.ToJsonString(false, false));
+                propertyChange1.NewValue.ShouldBe(newValue.ToJsonString(false, false));
+                propertyChange1.PropertyTypeFullName.ShouldBe(typeof(BlogEx).GetProperty(nameof(BlogEx.BloggerName)).PropertyType.FullName);
+
+                var entityChange2 = s.EntityChanges.Single(ec => ec.EntityTypeFullName == typeof(BlogPromotion).FullName);
+                entityChange2.ChangeType.ShouldBe(EntityChangeType.Created);
+                entityChange2.EntityId.ShouldBe(blog1Id.ToJsonString(false, false));
+                entityChange2.PropertyChanges.Count.ShouldBe(2);
+
+                var propertyChange2 = entityChange2.PropertyChanges.Single(pc => pc.PropertyName == nameof(BlogPromotion.BlogId));
+                propertyChange2.NewValue.ShouldNotBeNull();
+                propertyChange2.OriginalValue.ShouldBeNull();
+                propertyChange2.PropertyTypeFullName.ShouldBe(typeof(BlogPromotion).GetProperty(nameof(BlogPromotion.BlogId)).PropertyType.FullName);
+
+                var propertyChange3 = entityChange2.PropertyChanges.Single(pc => pc.PropertyName == nameof(BlogPromotion.AdvertisementId));
+                propertyChange3.NewValue.ShouldNotBeNull();
+                propertyChange3.OriginalValue.ShouldBeNull();
+                propertyChange3.PropertyTypeFullName.ShouldBe(typeof(BlogPromotion).GetProperty(nameof(BlogPromotion.AdvertisementId)).PropertyType.FullName);
+
+                return true;
+            };
+
+            _entityHistoryStore.Received().Save(Arg.Is<EntityChangeSet>(s => predicate(s)));
+        }
+
+        [Fact]
+        public void Should_Write_History_For_Owned_Entities_Of_Audited_Entities_Update()
+        {
+            // Blog is the owner of BlogEx has Audited attribute.
+            // Therefore, BlogEx should follow Blog and have entity history.
+
+            var newValue = "http://testblog1-changed.myblogs.com";
+            var originalValue = UpdateBlogUrlAndGetOriginalValue(newValue);
 
             Predicate<EntityChangeSet> predicate = s =>
             {
                 s.EntityChanges.Count.ShouldBe(1);
 
-                var entityChange = s.EntityChanges[0];
-                entityChange.ChangeType.ShouldBe(EntityChangeType.Updated);
-                entityChange.EntityId.ShouldBe(blog1Id.ToJsonString(false, false));
-                entityChange.EntityTypeFullName.ShouldBe(typeof(BlogEx).FullName);
-                entityChange.PropertyChanges.Count.ShouldBe(1);
+                var entityChangeBlog = s.EntityChanges[0];
```

Comment:
This naming and assignment is different from `Should_Write_History_For_Owned_Entities_Of_Audited_Entities_Create`.

Reply:
Fixed